### PR TITLE
Introducing SLE 12 SP4

### PR DESCRIPTION
**Pull request by Bischoff**
This PR adds the repos for SLE 12 SP4.

### CODE REVIEW

File: salt/mirror/minima.yaml [hunk 1]
```diff
@@ -89,15 +91,19 @@ http:
   - url: http://download.suse.de/ibs/SUSE:/Maintenance:/Test:/SLE-SERVER:/12-SP3:/x86_64/update
     archs: [x86_64]
 
+  # SLES 12 SP4 Test
+  - url: http://download.suse.de/ibs/SUSE:/Maintenance:/Test:/SLE-SERVER:/12-SP4:/x86_64/update
```
Comment (juliogonzalez) [inline salt/mirror/minima.yaml]: I am 99.99% sure this is SUSE Manager. Check with @mcalmer

Comment (juliogonzalez) [reply salt/mirror/minima.yaml]: But anyway it exists, so maybe we can use it safely already :-)

@Bischoff, check with @mcalmer, @ktsamis and @aliouliaki

Comment (moio) [reply salt/mirror/minima.yaml]: This repo contains SLE 12 SP4 updates in maintenance QA's queue, that is, all upcoming SLE fixes currently being tested. There is an option to enable those in sumaform in order to find out possible breaking changes before they are released to SUSE customers.

Comment (juliogonzalez) [reply salt/mirror/minima.yaml]: @moio, understood, thanks.

It seems I confused it with http://dist.suse.de/ibs/SUSE:/Maintenance:/Test:/SUSE-Manager-Server:/3.2:/x86_64/update/, which is the same but for SUSE Manager MUs, if I am not wrong.

**Comment by Bischoff** [comment]
It's now well tested and the PR seems stabilized. Time for attentive review before merging... @moio @juliogonzalez , final look please?

@mcalmer,  Julio asked me to confirm with you the change to `salt/repos/repos.d/SUSE-Manager-3.2-x86_64-Pool.repo`. I did it so it would be the same repo as for 3.0 and 3.1.

**Comment by Bischoff** [comment]
@moio @juliogonzalez : It's now well tested and the PR seems stabilized. Time for last review before merging... Thanks.

**Comment by mcalmer** [comment]
@Bischoff seems to be correct.

**Comment by Bischoff** [comment]
Thanks @mcalmer.

**Pull request merged**